docs/content/faq/how-can-i-deploy-a-docker-image.md

```diff
@@ -0,0 +1,43 @@
+# How can I deploy a Docker image to Flynn?
+
+Flynn supports running Docker containers from Docker registries. Note that currently the volumes of Docker apps are ephemeral and will be wiped when the instance reboots or is scaled down.
+
+Start by creating a config file for your Docker app, specifying the command to run, the job type, the init command, and any open ports.
+
+    {
+      "processes": {
+        "server": {
+          "cmd": ["memcached", "-u", "nobody", "-l", "0.0.0.0", "-p", "11211", "-v"],
+          "data": true,
+          "ports": [{
+            "port": 11211,
+            "proto": "tcp",
+            "service": {
+              "name": "memcached",
+              "create": true,
+              "check": { "type": "tcp" }
+            }
+          }]
+        }
+      }
+    }
+
+
+Flynn requires the Docker container's Docker registry url to install. The easiest way to get this is to install the container from the Docker registries, then run `docker images` to get the ID.
+
+    $ docker pull memcached
+    $ docker images --no-trunc|grep memcached
+    memcached            latest               8937d6d027a4f52b877aacd9faa68b8a89652d858845e1ea2cf7bc6a8306db00   13 days ago         132.4 MB
+
+Now, you can create the app via the Flynn CLI, create a release using the Docker registries url, and start the service by scaling it up.
+
+    # Create the app. Your app's name, in this case `memcached`, will be used for the app's hostname.
```

Comment:
Oooh, that's interesting. I hadn't tried this before, but I see now that Flynn lets me launch two apps with the same `service.name` key, and it looks like service discovery is grouping them into one service and that when I go to e.g. `memcached.discoverd` I can get routed to either of app1 or app2.